docs-gb/kubernetes/server-autoscaling.md

```diff
@@ -0,0 +1,124 @@
+# Server _Native_ Autoscaling
+
+## Overview
+Core 2 runs with long lived server replicas that can host multiple models (through multi-model serving, or MMS). These server replicas can be autoscaled natively to host these dynamically loaded models, allowing users to seamlessly optimize the infrastructure cost associated with their deployments. This document outlines the autoscaling policies and mechanisms that are available for autoscaling server replicas. These policies are designed to ensure that the server replicas are increased (scaled up) or decreased (scaled down) in response to changes in the number of model replicas that are required. In other words if a given model is scaled up, the system will scale up the server replicas to host these models. Similarly, if a given model is scaled down, the system will scale down the server replicas that are hosting these models.
+
+{% hint style="info" %}
+**Note**: Native autoscaling of servers is required in the case of MMS as the models are dynamically loaded and unloaded onto these server replicas. In this case Core 2 would autoscale server replicas according to changes to the model replicas that are required. This is in contrast to single-model autoscaling approach explained [here](kubernetes/hpa-rps-autoscaling.md) where server and model replicas are both autoscaled using. Server autoscaling can be used in the case of single-model serving, simplifying the autoscaling process as users would only needs to manage the scaling logic for model replicas, only requiring one HPA manifest.
+{% endhint %}
+
+## Requirements
+
+To enable autoscaling of server replicas, the following requirements need to be met:
+1. Setting `minReplicas` and `maxReplicas` in the `Server` CR. This will define the minimum and maximum number of server replicas that can be created.
+2. Setting `autoscaling.autoscalingServerEnabled` to `true` (default) during installation of Core 2. This will enable the autoscaling of server replicas.
+
+An example of a `Server` CR with autoscaling enabled is shown below:
+
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Server
+metadata:
+  name: mlserver
+  namespace: seldon
+spec:
+  replicas: 2
+  minReplicas: 1
+  maxReplicas: 4
+maxReplicas: 4
```

Comment:
duplicate `maxReplicas` (incorrect indent)?

Reply:
done in 
01e2376